protocols/file/handlers.go

```diff
@@ -14,21 +14,19 @@ import (
 	"github.com/libp2p/go-libp2p-core/network"
 )
 
-// Root handler for newFile protocol
-// It reads 2 bytes from the stream which determines the size of the request messeage
+// Handle implement root handler for newFile protocol
+// It reads 2 bytes from the stream which determines the size of the request message
```

Comment:
did we change "messeage" to "message" everywhere?

Reply:
Oh! This is already my mistake. This should not have happened.

Reply:
But, It seems to me that this is more correct. In the scope of this repository, this is the only error in accordance with "message".